src/views/AllNotice.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <section class="body">
+      <div>
+      <h1 class=heading>Notice Board</h1>
+      <!-- <Notice/> -->
+      <p style="padding:10px;text-align:center;">Want to Add a notice?<router-link style="color:red;" to="/addnotice"> Click here!</router-link></p>
+      <ul v-if="noticearr.length!==0">
+          <li v-for="not in noticearr" :key="not.id">
+              <div style="text-align:right">
+                  <span style="color:blue"><small>By: {{not.user}}</small></span>
+              </div>
+              <div style="text-align:left">
+                  <span><strong>{{not.notices}}</strong></span>
+              </div>
+              <div style="text-align:right">
+                  <span style="color:blue"><small>{{not.timestamp}}</small></span>
```

Comment:
Do we want the author name and the time to be at two different locations?
Either make it "Posted by foo on bar" at the top or at the bottom.